src/styles/card.css

```diff
@@ -0,0 +1,96 @@
+.card {
+  width: 200px;
+  height: 406px;
```

Comment:
p.8 of the [checklist](https://github.com/mate-academy/layout_product-cards/blob/master/checklist.md)